### ISSUE TITLE
Update documentation for setting up a dev environment in Windows

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#4322]
The setup script hasn't been working lately for Windows users, and it's always a moving target with gem and Ruby version support if a standalone Ruby environment is used on Windows.

However, it's now possible to set up a full working development environment using WSL. Documentation should be updated to reflect the WSL setup process (which is basically the Linux setup process), and most of the existing Windows-specific setup instructions and script should be removed.

**Comment by cyberShaw** [comment]
Can I work on this issue? I'm new to Open Source and would love to contribute :)

**Comment by ragesoss** [comment]
Yes, go for it.

**Comment by marklocklear** [comment]
@ragesoss going to spend some time looking at this. I'll focus on Windows 10 Pro and the documentation here: https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/docs/ubuntu_wsl_setup.md

**Comment by ragesoss** [comment]
@marklocklear thank you!

**Comment by masembebrian** [comment]
hey @ragesoss is it ok if i can take a look at this?

**Comment by ragesoss** [comment]
Yes. We've switched to pretty much only recommending WSL for Windows, but it's always welcome to have someone check out the current instructions and identify what is out of date or confusing.

**Comment by masembebrian** [comment]
hey @ragesoss, i wish to contribute to this.

**Comment by Abdulwahab03** [comment]
Hello @ragesoss, I would like to take this issue up, please any hint or guideline to get started will really help me

**Comment by ragesoss** [comment]
Sure. I suggest just trying it out, and report back with details about where it goes wrong.
